src/evergreen/tst.py

```diff
@@ -13,12 +13,10 @@
 class Logs(_BaseEvergreenObject):
     """Representation of test logs from evergreen."""
 
-    url = evg_attrib("url")
     line_num = evg_attrib("line_num")
-    url_raw = evg_attrib("url_raw")
-    url_lobster = evg_attrib("url_lobster")
```

Comment:
Removed url_lobster because it does not exist in evergreen test log object. lobster has been deprecated.